lib/handler.go

```diff
@@ -89,25 +89,30 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		// Gets the correct user for this request.
 		username, password, ok := r.BasicAuth()
-		zap.L().Info("login attempt", zap.String("username", username), zap.String("remote_address", remoteAddr))
+		// Removed the login attempt log to reduce log volume and focus on final verification results
+		// zap.L().Info("login attempt", zap.String("username", username), zap.String("remote_address", remoteAddr))
```

Comment:
This is fine, but please remove this line. It doesn't make sense to keep it there commented.

Reply:
Thanks for the feedback! I’ve removed the commented line as requested.

I initially left it there for comparison purposes, to highlight the change in log volume, but I agree that removing it entirely makes the code cleaner and more focused. Thanks again!